multiple_api_docs.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+function generateSubAPIDocs {
+  # $1 :- repo name
+  # $2 :- SUB project doc path
+  cd ${BUILD_DIR}/${1}
+  export SWAGGER_UI=""
+  export SWAGGER_SPEC_URL=""
+  export JAVADOC_PATH=""
+
+  # Setting environment variables for api docs
+  ENVVARS="$(python ${BASE}/modules/scripts/config.py)"
+  echo -e "\n${ENVVARS}\n" >> ${LOGFILE}
+  eval ${ENVVARS}
+
+  generateAPIDocs ${BUILD_DIR}/${1} ${BUILD_DIR}/_build/html/${1}/${2}
+
+}
+
+IFS=',' read -ra SUBPROJECT_URLS_ARRAY <<< "${SUBPROJECT_URLS}"
+IFS=',' read -ra SUBPROJECT_DOCPATHS_ARRAY <<< "${SUBPROJECT_DOCPATHS}"
+
+SUBPROJECT_DIRS_ARRAY=()
+
+for ((i=0;i<${#SUBPROJECT_URLS_ARRAY[@]};i++)); do
+  SUB_URL="${SUBPROJECT_URLS_ARRAY[$i]}"
+  SUB_DOCPATH="${SUBPROJECT_DOCPATHS_ARRAY[$i]}"
+  SUB_URL_SPLIT=(${SUB_URL//// })
+  SUB_REPONAME=(${SUB_URL_SPLIT[3]//./ })
+  generateSubAPIDocs ${SUB_REPONAME} ${SUB_DOCPATH}
+done
```

Comment:
I'm not talking about the loop. What I'm saying is that the entire block from `IFS ...` in the two script is same except the function call. It would be better if that code repetition can be avoided.